tests/next_tests/definitions.py

```diff
@@ -129,26 +128,36 @@ class ProgramFormatterId(_PythonObjectIdMixin, str, enum.Enum):
 # Common list of feature markers to skip
 COMMON_SKIP_TEST_LIST = [
     (REQUIRES_ATLAS, XFAIL, BINDINGS_UNSUPPORTED_MESSAGE),
+    (USES_DYNAMIC_OFFSETS, XFAIL, UNSUPPORTED_MESSAGE),
```

Comment:
I would prefer to not have `USES_DYNAMIC_OFFSETS` in `COMMON_SKIP_TEST_LIST`. The DaCe ITIR backend supports it.